servers/republik/modules/crowdfundings/lib/scheduler/deactivate.js

```diff
@@ -43,32 +36,41 @@ const deactivate = async (args, { pgdb, mail: { enforceSubscriptions } }) => {
   debug({
     cancelledEndDate: cancelledEndDate.toDate(),
     uncancelledEndDate: uncancelledEndDate.toDate(),
-    memberships: memberships.length
+    memberships: memberships.length,
+    runDry
   })
 
+  if (runDry) {
+    debug('dry run')
+    return
+  }
+
   await Promise.each(
     memberships,
     async membership => {
-      const transaction = await pgdb.transactionBegin()
       debug({ membership: membership.id })
+      if (runDry) {
+        return
+      }
```

Comment:
remove, you already exit on L43.

Reply:
Removed [earlier exit](https://github.com/orbiting/backends/pull/176/commits/44675f13c7915faa5fe3d4cca694217f7c03a65a), keeping this one to understand which memberships would be deactivated on a debug level.